src/robot/parsing/model/statements.py

```diff
@@ -450,6 +457,16 @@ def _add_error(self, error):
 class IfHeader(Statement):
     type = Token.IF
 
+    @classmethod
+    def from_params(cls, condition, indent='    ', separator='    ', eol='\n'):
```

Comment:
@bhirsz maybe default indent, separator and eol could be extracted to constants like:
```
def from_params(cls, condition, indent=INDENT, separator=SEPARATOR, eol=UNIX_EOL):
```
where INDENT, SEPARATOR and UNIX_EOL:
```
INDENT='    '
SEPARATOR='    '
UNIX_EOL='\n'
```

Reply:
I'm not sure about this. Problems:
- Seeing actual values directly would have its benefits.
- Because default values are bound when functions are created, changing constant values won't have any effect anyway.
- `UNIX_EOL` is a confusing name because `\n` is a generic newline in Python and converted to OS specific newline when saving. Passing `\r\n` as the EOL would in most cases cause wrong results on Windows.

The above said, repeating `'    '` everywhere is annoying as well and avoiding that may be a good idea anyway. `UNIX_EOL` should nevertheless be just `EOL`. Instead of `SEPARATOR` and `INDENT` we could also use `FOUR_SPACES` to make the actual value explicit. It's longer than the alternatives, though.

Reply:
@pekkaklarck my idea was to provide constans as default values, unless function is invoked with other values, just to avoid repeating and possible typos when using `'    '`. I agree that `EOL` is way better. I didn't know that python was replacing that based on OS used. Good to know. `FOUR_SPACES` is descriptive, so even tough it's longer name it is better one.

Reply:
Ok, let's go with `FOUR_SPACES` and `EOL`.